gae/js/app/views/sitecaptain.js

```diff
@@ -68,11 +68,18 @@ define(
                 m.destroy();
                 this.render();
             },
+            before_save: function(e){
+                e.preventDefault();
+                var id_value = Number(this.$el.find('select').val());
+                var choices = this.form.fields.models[1].changed.options;
```

Comment:
The [1] looks brittle - meaning that if something changes about that array this will break suddenly.  Let's find another way to refer to that list of options.

Reply:
Also... I 'm getting a console error sometimes which may be related.  Seems that this.filtered_captain is not defined. Sometimes. But sometimes it works.  

jquery.js:2 jQuery.Deferred exception: Cannot read property 'label' of undefined TypeError: Cannot read property 'label' of undefined
    at Object.<anonymous> (https://issue357-dot-rebuildingtogethercaptain-hrd.appspot.com/js/app/views/sitecaptain.js:82:65)
    at j (https://issue357-dot-rebuildingtogethercaptain-hrd.appspot.com/js/lib/jquery.js:2:29999)
    at k (https://issue357-dot-rebuildingtogethercaptain-hrd.appspot.com/js/lib/jquery.js:2:30313) undefined